spack/package.py

```diff
@@ -155,12 +162,10 @@ def url_for_version(self, version):
     conflicts("+netcdf3", when="+netcdf")
     conflicts("+openmp", when="%apple-clang")
 
-    depends_on("wget", type=("build"), when="@:3.1 +netcdf4")
-    depends_on("wget", type=("build"), when="@:3.1 +netcdf")
     depends_on("ip@5.1:", when="@develop +ipolates")
     depends_on("libaec@1.0.6:", when="@3.2: +aec")
     depends_on("netcdf-c", when="@3.2: +netcdf4")
-    depends_on("netcdf-c", when="@3.2: +netcdf")
+    depends_on("netcdf-c", when="@3.4: +netcdf")
```

Comment:
Technically the `@3.4:` isn't needed since that variant is undefined for earlier versions anyway, but, it's not hurting anything either.

Reply:
I just removed it.